src/styles/admin-core/_content-block-form.scss

```diff
@@ -30,6 +30,11 @@
 		}
 	}
 
+	.c-button__icon.o-svg-icon svg {
+		height: 1.8rem;
```

Comment:
Use a variable for this so you can reuse it for all other values related to this value: $button-size: 1.8rem;